### DISCUSSION TITLE
Ngnix vs Cloundfront 뭐가 다른건가요??

### DISCUSSION THREAD

**Discussion by inyeong-kang** [woowacourse-teams/2023-votogether#497] [질문]
dd

**Comment by chsua** [comment]
[참고자료](https://jw910911.tistory.com/148)

위 불로그에서는 
1. AWS - EC2 + WebServer(Nginx) 를 활용한 배포
2. AWS - CloudFront + S3를 활용한 배포

두 가지를 비교하고 있습니다. 

크게 장단점이 
- EC2 + WebServer(Nginx) 를 활용한 배포: 하나의 웹루트를 통해 여러 Demo들을 관리할 수 있기 때문에 관리가 용이하나 전 세계에 지역에 있는 Client가 EC2 서버로 정적인 페이지를 요청할 때 latency(응답시간)가 더 오래 걸릴 수 있음
- CloudFront + S3를 활용한 배포: 고가용성 및 내구성을 보장하고 여러 나라에서 서비스를 할 때 안정적인 속도를 보장하지만 러닝커브 존재

라우터와 관련된 내용이나, 서브 디렉터리 등에 대한 이야기가 있는데 주요한 내용은 위와 같습니다. 
내용이 많아서 들어가서 읽어보시는게 좋을 것 같습니다.
참고하면 좋을 것 같습니다.
(다른 방식도 있지만 둘만 고려대상이니 생략하겠습니다.)
